packages/expo-updates/src/statemachine/UpdatesStateMachine.ts

```diff
@@ -67,6 +67,7 @@ const checkCompleteUnavailableAction = assign({
 });
 
 const checkErrorAction = assign({
+  // @ts-ignore: FIXME
```

Comment:
Hi @douglowder, I'm not too familiar with this part of our code, do you mind taking a look at this? Seems that `xstate` is not inferring things correctly

Reply:
Prefer `ts-expect-error`

Reply:
Good call, updated